tests/threshold/threshold-config-rate-filter-alert-flow/threshold.config

```diff
@@ -0,0 +1 @@
+rate_filter gen_id 1, sig_id 1000001, track by_flow, count 1, seconds 60, new_action alert, timeout 1000
```

Comment:
Would like a comment stating that `seconds 60` does not matter, could be `seconds 1` as we count only one event

Reply:
updated the test to use count 3

Reply:
even better